tests/db/pg_client.py

```diff
@@ -1,16 +1,47 @@
-import os
-
 from dotenv import load_dotenv
 import psycopg2
+from psycopg2._psycopg import cursor, connection
+from duneapi.types import DuneQuery
+from psycopg2.extras import RealDictCursor
 
 
 def connect():
     load_dotenv()
     conn = psycopg2.connect(
-        host=os.environ["POSTGRES_HOST"],
-        port=os.environ["POSTGRES_PORT"],
+        host="localhost",
+        port=5432,
         database="postgres",
         user="postgres",
         password="postgres",
     )
     return conn
+
+
+def populate_db() -> tuple[connection, cursor]:
```

Comment:
nit: maybe connect_and_populate_DB weould be better? I was pretty surprised about the return type given that name.

Reply:
Note that the reason the return type must have both cursor and connection, so that we can close the connection in the `teardown()`. I will make the suggested name change here and add a comment on this matter.